core/unit_test/view/TestBasicViewMDSpanConversion.cpp

```diff
@@ -21,27 +21,28 @@
 static_assert(
     std::is_convertible_v<
         Kokkos::View<long long ****, Kokkos::LayoutRight, Kokkos::Serial>,
-        Kokkos::Impl::BasicView<long long, Kokkos::dextents<size_t, 4>,
+        Kokkos::Impl::BV::BasicView<long long, Kokkos::dextents<size_t, 4>,
```

Comment:
Consider having a using-declaration `using Kokkos::Impl::Foo::BasicView;` and use unqualified `BasicView` in this file.

Reply:
yeah I can do that.

Reply:
Done